### DISCUSSION TITLE
Popover button on top of parent with a click handler doesn get the event

### DISCUSSION THREAD

**Discussion by cubecleveland** [ariakit/ariakit#922] [Questions] [answered]
Hello, we have a popover inside a div that has its own click handler.

The popver is the 2nd inline to recieve the event propgation.
so the first one is the div then the popver button.
To comapre if we put a simple <button> in the same div our button get the evnt first the propagate to the div....
Is there anyreason for this or something we are missing.?
Is that the expected behaviour ?

**Comment by diegohaz** [comment]
Could you please provide a CodeSandbox reproducing the problem?

**Comment by cubecleveland** [comment, marked as answer]
i will work on one.

**Reply by cubecleveland** [reply]
this was resolved
